### PR TITLE
Adding affiliates document

### PR DESCRIPTION
**Pull request by dgkf**
Adding a small outline of affiliated initiatives with a bit of discussion about overlap and differences in remit. 

The goal is that this will help build familiarity of our partners for people who might not be familiar with these other teams to know how and when they might be strong assets to our project.

@kkmann - Would you mind reading through to weigh in on whether this is helpful or suggest other info that might help.

### CODE REVIEW

File: notes/affiliates.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+## [`pharmaverse`](https://github.com/pharmaverse)
+
+> A curated, opinionated, pharma stack of open source R packages to enable
+> clinical reporting (from CRF to eSubmission) backed by a community of
+> passionate individuals and organisations committed to co-creating efficiency
+> in our mission to improve health.
+
+In other words, a set of R packages to achieve common submission workflows.
+Although these packages intend to meet a level of quality that would make them
+suitable for developing submission analysis.
```
Comment (kkmann) [inline notes/affiliates.md:59]: pharmaverse currently does not strive to build their own infrastructure, right? More of a 'tidyverse' appraoch ? Tagging @epijim here.

File: notes/affiliates.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+initiatives that each support the grand vision of streamlining the use of R in
+a regulated setting.
+
+In the past this has involved endorsing a white paper that details how R might
+be used in a regulated setting, building tools to support individual R package
+evaluation, and offering an open forum for companies to share their approaches.
+
+To further support this, the next phase of the R Validation Hub initiatives have
+been focused on lifting these activities out of individual organizations and
+into an open, shared context. This is where we come in!
```
Comment (kkmann) [inline notes/affiliates.md:48]: @dgkf do we have anyone from GSK onboard already?

**Comment by kkmann** [comment]
Looks amazing, thank you! I think the crucial thing is making sure that we do not overlap with r val hub (we don't we are a spin off) and the pharmaverse. R consortium repo WG could be foundational if we indeed decide that hosting our own CRAN like repo is the way forward.

File: notes/affiliates.md (hunk 1 above)
Comment (dgkf) [reply notes/affiliates.md:59]: I believe that's correct. If my memory serves, they reinforced this when talking about their scope on the call with their project leads.

File: notes/affiliates.md (hunk 2 above)
Comment (dgkf) [reply notes/affiliates.md:48]: I don't see anyone in any of our invites with a GSK email. I don't think we have anyone whose reached out yet.

**Comment by dgkf** [comment]
@kkmann - Any changes you'd want to make? If not, I'd err toward pulling this in and we can always make adjustments from there.

**Review by kkmann — APPROVED** [review]
👍🏻

**Pull request merged**